packages/wrangler/src/__tests__/tail.test.ts

```diff
@@ -167,10 +167,27 @@ describe("tail", () => {
   /* Printing */
 
   it("logs incoming messages", async () => {
-    await runWrangler("tail test-worker");
-    const message = serialize(generateMockEventMessage());
-    api.ws.send(message);
-    expect(std.out).toMatch(deserializeToJson(message));
+    await runWrangler("tail test-worker --format json");
+
+    const requestEvent = generateMockRequestEvent();
+    const requestMessage = serialize(
+      generateMockEventMessage({ event: requestEvent })
+    );
+    api.ws.send(requestMessage);
+    expect(std.out).toMatch(deserializeToJson(requestMessage));
+
+    const scheduledEvent = generateMockScheduledEvent();
+    const scheduledMessage = serialize(
+      generateMockEventMessage({
+        event: scheduledEvent,
+      })
+    );
+    api.ws.send(scheduledMessage);
+    expect(std.out).toMatch(deserializeToJson(scheduledMessage));
+  });
+
+  it("omits non-essential information when pretty-printing", async () => {
+    await runWrangler("tail test-worker --format pretty");
```

Comment:
I think we're missing some expectation checks here?

Reply:
yeah i accidentally committed more files than i had meant to, fixing